assets/js/modules/adsense/components/dashboard/AdBlockingRecoveryWidget.test.js

```diff
@@ -42,14 +42,11 @@ import {
 	SITE_STATUS_READY,
 } from '../../util';
 import { getWidgetComponentProps } from '../../../../googlesitekit/widgets/util';
-import { WEEK_IN_SECONDS, DAY_IN_SECONDS } from '../../../../util';
 
 describe( 'AdSenseConnectCTA', () => {
 	let registry;
-	const timestampThreeWeeksAgo =
-		Math.floor( Date.now() / 1000 ) - WEEK_IN_SECONDS * 3;
-	const timestampLessThanThreeWeeksAgo =
-		timestampThreeWeeksAgo + DAY_IN_SECONDS;
+	const timestampThreeWeeksAgo = '1685576083'; // Unix timestamp for 2023-06-01
+	const timestampLessThanThreeWeeksAgo = '1685662483'; // Unix timestamp for 2023-06-02
```

Comment:
Let's make things a little clearer by defining the reference date here and renaming these constants.

Also - I realise these example dates were what I pasted in my suggestion but they are actually incorrect as they're _~2_ weeks from the reference date, not 3. I suspect this is why you've ended up adding a second reference date call, which shouldn't be needed. As a result I've updated the reference date value in my suggestion too. Please double check all of these date values.

Finally, as the timestamps, according to the comments are supposed to represent days, let's round them to the day i.e. midnight.

```suggestion
	const referenceDate = '2023-06-22';
	const timestampThreeWeeksPrior = '1685577600'; // Unix timestamp for 2023-06-01.
	const timestampLessThanThreeWeeksPrior = '1685664000'; // Unix timestamp for 2023-06-02.
```